cmd/kepctl/cmd/promote.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"k8s.io/enhancements/pkg/kepctl"
+)
+
+// TODO: Struct literal instead?
```

Comment:
I prefer no struct and have it be local to a function. https://github.com/kubernetes-sigs/zeitgeist/blob/master/buoy/commands/exists.go#L30 and this lets you reuse common flag defs like https://github.com/n3wscott/bujo/blob/master/pkg/commands/get.go#L89